lib/elixir_analyzer/test_suite/german_sysadmin.ex

```diff
@@ -0,0 +1,37 @@
+defmodule ElixirAnalyzer.TestSuite.GermanSysadmin do
+  @moduledoc """
+  This is an exercise analyzer extension module for the concept exercise German Sysadmin
+  """
+
+  use ElixirAnalyzer.ExerciseTest
+
+  assert_no_call "doesn't convert anything to a string" do
+    type :essential
+    called_fn name: :to_string
+    comment ElixirAnalyzer.Constants.german_sysadmin_no_strings()
+  end
+
+  assert_no_call "doesn't convert anything to a charlist" do
+    type :essential
+    called_fn name: :to_charlist
+    comment ElixirAnalyzer.Constants.german_sysadmin_no_strings()
+  end
+
+  assert_no_call "doesn't use any string functions" do
+    type :essential
+    called_fn module: String, name: :_
+    comment ElixirAnalyzer.Constants.german_sysadmin_no_strings()
+  end
+
+  assert_no_call "doesn't create binaries from character codes" do
+    type :essential
+    called_fn name: :<<>>
+    comment ElixirAnalyzer.Constants.german_sysadmin_no_strings()
+  end
+
+  assert_call "using case is required" do
+    type :essential
+    called_fn name: :case
+    comment ElixirAnalyzer.Constants.german_sysadmin_use_case()
```

Comment:
Nice! This is a good example of how using `assert_call` works better than `feature` in some cases

Reply:
Yup! I was really happy to realize it can be used like this.